src/Majora/Framework/Loader/Bridge/Form/Type/EntityCollectionType.php

```diff
@@ -0,0 +1,124 @@
+<?php
+
+namespace Majora\Framework\Loader\Bridge\Form\Type;
+
+use Majora\Framework\Loader\Bridge\LoaderBridgeInterface;
+use Majora\Framework\Model\EntityCollection;
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\Extension\Core\Type\ChoiceType;
+use Symfony\Component\OptionsResolver\Options;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+/**
+ * Class EntityCollectionType.
+ */
+class EntityCollectionType extends AbstractType
+{
+    /**
+     * @var array
+     */
+    private $loaders = [];
+
+    /**
+     * {@inheritdoc}
+     */
+    public function configureOptions(OptionsResolver $resolver)
+    {
+        parent::configureOptions($resolver);
+
+        $choices = function (Options $options) {
+            if ($this->hasLoader($options['loader_alias']) === false) {
+                throw new \RuntimeException(sprintf('Loader with alias "%s" not found.', $options['loader_alias']));
+            }
+
+            $loader = $this->getLoader($options['loader_alias']);
+
+            if (method_exists($loader, $options['loader_method']) === false) {
+                throw new \RuntimeException(
+                    sprintf('Method "%s" not found in "%s".', $options['loader_method'], get_class($loader))
+                );
+            };
+
+            $data = $loader->{$options['loader_method']}();
```

Comment:
Choices could be implemented as "choices" option normalization using OptionsResolver process : http://symfony.com/doc/current/components/options_resolver.html#default-values-that-depend-on-another-option